core/dbt/task/test.py

```diff
@@ -96,8 +97,8 @@ def print_result_line(self, result):
         model = result.node
 
         fire_event(
-            LogTestResult(
-                name=self.describe_node_name(),
```

Comment:
Why was the change made to stop using self.describe_node_name?

Reply:
oh, it's an issue with merging main branch, I'll check.

Reply:
I've restored it but test is failed.. sorry for the confusion.
I'll fix it quickly!